### ISSUE TITLE
Scope UI-TARS finetuning setup

### ISSUE THREAD

**Issue by Locke0** [ManifoldRG/software-control#91]
- LoRA / no LoRA
- VeOmni or VeOmni + Unsloth
- investigate OS-Atlas
    - data mixture
- investigate Jedi
    - data mixture
- think about our data mixture based on above investigations

**Comment by Locke0** [comment]
Tracking here: https://coda.io/d/Software-Control-Models-Knowledge-Base_dzujAXxemDw/Scope-fine-tuning-setup_subEBdzO#_luJNmZos

**Comment by Locke0** [comment]
Final decision on the small-scale experiment:
- create 3 dataset split variants with 1k, 3k, 10k examples:
    - JEDI dataset only
    - JEDI + custom augmented web data
    - augmented web data only
- SFT with LoRA on UI-TARS-1.5 like Fintor's setup for UI-TARS on OS-Atlas dataset (which improves its screenspot-v2 and pro performance)
- evaluate on ~500-1k screenspot-pro and/or OSWorld-G validation set.
- analyze the eval result

**Comment by Locke0** [comment]
Marking as complete, the remaining details such finetuning infra, cloud infra, etc will be tracked in the corresponding implementation tickets
